pkg/names/names.go

```diff
@@ -134,6 +134,17 @@ const OpenShiftComponent = "openshift.io/owning-component"
 // ClusterNetworkOperatorJiraComponent is the jira component name for the cluster-network-operator
 const ClusterNetworkOperatorJiraComponent = "Networking / cluster-network-operator"
 
+// Status condition types of network.config for live migration
+const (
+	NetworkTypeMigrationAnnotation         string = "network.openshift.io/live-migration"
```

Comment:
This is not a condition, let's move it out from this block. 
Additionally please add a doc to each variable.